kwalitee/cli.py

```diff
@@ -116,3 +134,34 @@ def generate_tool_json(ctx, output=None):
         Path(output).write_text(json_str)
     else:
         print(json_str)
+
+
+@cli2.command('validate', help='''
+
+    Validate created JSON files.
+
+''')
+@click.option('-f', '--file',
+    type=click.Choice(['repos,json', 'ocrd_all_releases.json'], case_sensitive=True),
+    help="The file to be validated. Defaults to repos.json if not given.")
+def json_validate(file=None):
+    if not file:
+        file_to_validate = 'repos.json'
+    else:
+        file_to_validate = file
+
+    with open(file_to_validate) as f:
+        instance = json.load(f)
+
+    with open('schema/' + file_to_validate) as s:
+        schema = json.load(s)
+
+    _inform_of_result(JsonValidator(schema).validate(instance, schema))
```

Comment:
`validate` is a static class method, so no need to instantiate twice:

```suggestion
    _inform_of_result(JsonValidator.validate(instance, schema))
```